python/cudf/cudf/tests/test_column.py

```diff
@@ -31,12 +31,13 @@
 @pytest.fixture(params=dtypes, ids=dtypes)
 def pandas_input(request):
     dtype = request.param
-    rng = np.random.default_rng()
+    rng = np.random.default_rng(seed=None)
```

Comment:
I know this is a no-op change, but is there a reason you're not setting the seed for this rng?

Reply:
Nope, I think we must seed it. Updated.